cells/lib/ops/mkStandardOCI.nix

```diff
@@ -62,7 +65,7 @@ in
       else operable;
 
     setupLinks = cell.ops.mkSetup "links" [] ''
-      mkdir -p $out/bin
+      mkdir -p $out/{bin,tmp}
```

Comment:
Should this be larger discussion about what a default container should look like? I have more ideas we could add, but I opted to leave it up to the end user to implement.

Reply:
potentially, but I feel like `/tmp` is fairly harmless on its own

Reply:
Yeah, I suppose it just feels out of place, given the intent of the commit message. I guess I was suggesting maybe parsing it out into a separate discussion that addresses the more significant issue of having a more helpful "base image" generated by the function.

Reply:
> more significant issue of having a more helpful "base image" generated by the function.

Maybe we use `preset=false` to disable some configurations and keep a primitive image.

Reply:
I can just remove the commit that adds /tmp for now and do a separate change-set for that. Good idea

Reply:
> Maybe we can use `preset=false or pure` to disable some configurations and keep a primitive image.

This or something similar is likely what we should do in the end. I opened #274 as a draft for now to hash that out further.